### PR TITLE
Bikeshed improvements related to CSS Text Decorations 3 property values

### PR DESCRIPTION
**Pull request by SebastianZ**

### CODE REVIEW

File: css-text-decor/Overview.bs [hunk 1]
```diff
@@ -211,36 +187,14 @@ Text Decoration Lines: the 'text-decoration-line' property</h3>
...
-      <th>Computed&#160;value:
-      <td>the computed color
-    <tr>
-      <th>Animatable:
-      <td>as <a href="https://www.w3.org/TR/css3-transitions/#animtype-color">color</a>
-  </table>
+  <pre class="propdef">
+  Name: text-decoration-color
+  Value: <<color>>
+  Initial: ''currentColor''
```
Comment (tabatkins) [inline css-text-decor/Overview.bs]: Don't link the value here; it'll result in visible quotes which is confusing.

File: css-text-decor/Overview.bs [hunk 2]
```diff
@@ -860,77 +688,29 @@ Emphasis Mark Style: the 'text-emphasis-style' property</h3>
...
-      <th>Computed&#160;value:
-      <td>as specified
-    <tr>
-      <th>Animatable:
-      <td>as <a href="https://www.w3.org/TR/css3-transitions/#animtype-color">color</a>
-  </table>
+  <pre class="propdef">
+  Name: text-emphasis-color
+  Value: <<color>>
+  Initial: ''currentColor''
```
Comment (tabatkins) [inline css-text-decor/Overview.bs]: Omit quotes here.

File: css-text-decor/Overview.bs [hunk 3]
```diff
@@ -860,77 +688,29 @@ Emphasis Mark Style: the 'text-emphasis-style' property</h3>
...
+  Inherited: yes
+  Animation type: <a href="https://www.w3.org/TR/css3-transitions/#animtype-color">color</a>
+  </pre>
 
   <p>This property specifies the foreground color of the emphasis marks.</p>
 
   <p class="note">
-    The <dfn value for=text-emphasis-color>currentcolor</dfn> keyword computes to itself
+    The <dfn value for=text-emphasis-color>currentColor</dfn> keyword computes to itself
```
Comment (tabatkins) [inline css-text-decor/Overview.bs]: All-lowercase is correct (CSS keywords can have any casing you want, but we purposely use all-lowercase for consistency).

Go ahead and remove the `<dfn>` entirely, and instead just autolink the value as `''currentcolor''`.  Dfns don't belong in Notes, and we're not actually defining this value here; it's defined by the Color spec.

**Comment by tabatkins** [comment]
:+1: after nits

**Comment by SebastianZ** [comment]
Addressed the comments in https://github.com/w3c/csswg-drafts/pull/233/commits/4b1016557d33c655545f380c1ba423395a34b086.

Sebastian

**Pull request merged**